plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -313,6 +316,14 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			return nil
 		}
 		return nil
+	case runTraceAgain:
```

Comment:
could it support rerun a live traceflow too?

Reply:
any feedback on this comment?

Reply:
No, it can't since a Live Traceflow doesn't create a Traceflow Request.

Reply:
It creates a Traceflow request: https://github.com/antrea-io/antrea/blob/main/plugins/octant/cmd/antrea-octant-plugin/traceflow.go#L289.

Reply:
I'm adding functionality to be able to rerun live traceflows. My mistake, I didn't see that line.